src/App.tsx

```diff
@@ -2,35 +2,40 @@ import { Suspense } from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import { QueryParamProvider } from 'use-query-params';
 import { ReactRouter5Adapter } from 'use-query-params/adapters/react-router-5';
-import { AppFooter, AppHeader, DerivIframe } from '@/components';
-import { useRedirectToOauth } from '@/hooks';
+import { AppFooter, AppHeader, DerivIframe, ErrorBoundary } from '@/components';
+import { useRedirectToOauth, useTrackjs } from '@/hooks';
 import AppContent from '@/routes/AppContent';
 import { initializeI18n, TranslationProvider } from '@deriv-com/translations';
 import { Loader, useDevice } from '@deriv-com/ui';
 
-const { VITE_CROWDIN_BRANCH_NAME, VITE_PROJECT_NAME, VITE_TRANSLATIONS_CDN_URL } = import.meta.env;
+const { VITE_CROWDIN_BRANCH_NAME, VITE_PROJECT_NAME, VITE_TRANSLATIONS_CDN_URL } = process.env;
```

Comment:
I dont think that this should be changed.

Reply:
the reason was that I was getting ts error in my test files. so just updated it like this. also I've updated vite config . is there any issues with this approach? 🤔